### PR TITLE
feat: appHeader implementation

### PR DESCRIPTION
**Pull request by kibo-chandradeeptalaha**

### CODE REVIEW

File: components/layout/AppHeader/KiboHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+
+const StyledToolbarNav = styled(Toolbar)(() => ({
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'flex-end',
+  gap: 50,
+}))
+
+interface KiboHeaderProps {
+  navLinks: {
```
Comment (kibo-kevinwatts) [inline components/layout/AppHeader/KiboHeader.tsx:29]: pass these navlinks to the hamburger navigation menu

File: components/layout/AppHeader/KiboHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+                >
+                  <SearchBar searchTerm="" onSearch={handleSearch} showClearButton />
+                </Box>
+                <Box
+                  sx={{
+                    order: { xs: 4, md: 2 },
+                    marginLeft: 'auto',
+                  }}
+                >
+                  <HeaderAction title="Find a Store" subtitle="View All" icon={FmdGoodIcon} />
```
Comment (kibo-kevinwatts) [inline components/layout/AppHeader/KiboHeader.tsx:202]: use translations for the three header actions

**Review by kibo-kevinwatts — CHANGES_REQUESTED** [review]
looks good!

File: components/layout/AppHeader/KiboHeader.tsx (hunk 1 above)
Comment (kibo-chandradeeptalaha) [reply components/layout/AppHeader/KiboHeader.tsx:30]: Hamburger menu code is commented for now. Will add that once the codes get merged.

File: components/layout/AppHeader/KiboHeader.tsx (hunk 2 above)
Comment (kibo-chandradeeptalaha) [reply components/layout/AppHeader/KiboHeader.tsx]: Done

File: components/common/HeaderAction/HeaderAction.tsx [hunk 3]
```diff
@@ -28,7 +28,13 @@ const HeaderAction = (props: HeaderActionProps) => {
   const { title, subtitle, onClick, badgeContent, mobileIconColor = 'white' } = props
   const Icon = props.icon
   return (
-    <Box display="flex" alignItems="center" onClick={onClick}>
+    <Box
+      display="flex"
+      alignItems="center"
+      marginX={2}
```
Comment (amolpawar-kibo) [inline components/common/HeaderAction/HeaderAction.tsx:34]: can we combine marginX &  SX?

**Review by kibo-kevinwatts — APPROVED** [review]

**Pull request merged**